_pages/en/strategy-alternate-model.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: default
+title: Alternate IT Solution Delivery Model
+ref: alternate-delivery-model
+lang: en
+status: posted
+sections: Strategies
+permalink: /strategy-alternate-model.html
+---
+<!-- markdownlint-disable MD033 -->
+<!-- the below cSpell statement says to ignore any text between HTML tags. E.g. ignore <table> -->
+<!-- cSpell:ignoreRegExp /^\<.+\>$/ -->
+## {{ page.title }} <!-- omit in toc -->
+
+***DRAFT***
+
+The purpose of this strategy is to provide IITB with an alternative IT solution delivery model.
+One that will enable ESDC to become more agile, open, and user-focused, as expected by the [Government of Canada's Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html)
+
+The Strategy includes:
+
+1. A **guiding policy**, setting automatic decisions when this alternative IT solution delivery model is  used
+2. A **coherent set of actions** (an action plan), to make the alternative IT solution delivery model operational
+3. Metrics to **measure the strategy's success** and compare it with the conventional delivery model
+
+The **business case** for this strategy is provided in [Appendix A](#appendix-a---business-case-diagnostic)
+<!-- markdownlint-disable MD001 -->
+##### Table of Content <!-- omit in toc -->
+<!-- markdownlint-enable MD001 -->
+- [Guiding Policy](#guiding-policy)
+- [Coherent set of actions](#coherent-set-of-actions)
+- [Measuring the Strategy's success](#measuring-the-strategys-success)
+- [Appendix A - Business Case (Diagnostic)](#appendix-a---business-case-diagnostic)
+- [Appendix B - Traceability Matrix](#appendix-b---traceability-matrix)
+- [Appendix C - References](#appendix-c---references)
+
+## Guiding Policy
+
+The following policy reflects the decision adopted by the **CIO of ESDC** when using this Alternative Solution Delivery Model.
+Each policy statement is a declaration of that decision and received the endorsement of its associated area of governance body (endorsements not yet obtained, see section coherent set of actions).
+
+***Governance and Compliance***
+
+1. IT Product teams have the authority to make changes to their IT products directly in production
```

Comment:
Perhaps phrasing like "IT teams have the authority, with proper automation in place, to have changes be made in production environments."

This phrasing may cause people to think people are logging into production servers and tinkering with things manually -- which shouldn't be the case. At least, should be explicitly permitted or promoted, even if it has to happen in unfortunate urgent situations.